source/includes/aws/_clusters.md

```diff
@@ -0,0 +1,140 @@
+`numberOfNodes` <br/> *integer* | The total number of nodes in the cluster.
+`name` <br/> *string* | The name of the cluster.
+`caCert` <br> *string* | The certificate authority data for the cluster.
+<!-------------------- RETRIEVE A CLUSTER -------------------->
+
+#### Retrieve a cluster
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v2/services/aws/test-area/clusters/test-cluster-1"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": {
+    "id": "test-cluster-1",
+    "name": "test-cluster-1",
+    "status": "ACTIVE",
+    "endpoint": "https://2CEF4912C1CF0F97B068143243A7720B.gr7.us-east-1.eks.amazonaws.com",
+    "creationTimestamp": "2022-06-30 12:57:11",
+    "addressRange": "10.100.0.0/16",
+    "vpcName": "vpc-0c11473c060163620",
+    "subnetNames": [
+        "subnet-0788e84e9f74187fc",
+        "subnet-07fbc2fe11acae389",
+        "subnet-00d68e63b69ebebbc",
+        "subnet-06fcc57288a2df042",
+        "subnet-04fd0e04992598e85"
+    ],
+    "totalMemory": 2.0,
+    "totalVcpus": 4,
+    "numberOfNodes": 2,
+    "version": "1.22",
+    "caCert": "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUMvakNDQWVhZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwKY201bGRHVnpNQjRYRFRJeU1EY3hNekEwTlRBeE5Wb1hEVE15TURjeE1EQTBOVEF4TlZvd0ZURVRNQkVHQTFVRQpBeE1LYTNWaVpYSnVaWFJsY3pDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTWx5CmZ5QmZhTThUZENLSzJuZ7VlK3RIVC9EZjBPL2h4SEMrUi9LUU0rL3dTckNoRTRHdGdNT1hHc2tqbCsyWjFEWSsKbnRqTVlGOXVFaS9KWVlMaHB5blhkeCtZajd9bXVFK3JoQUhvQ0Y3dzg2VXN1Z1pITkFDcmE1YzNOWkpmWUdiRApqNkVxU0l0VUhVUG5NNkdnRlkwam9DcXlmUlI0Mk9JdWVQTjlDS2UrMzJyTjY0akVsZDc1THdQS2U2RWhRdllmClUyLzcyRFFJTHMyRHJXbGpOdGZmdi9DbGxQZy85TDVMSHpWdWRXNDcxeFBOZVZnQ3hOTlFMTlRaV1J1UFc4TGcKYnFqUWUvTEc4aHlQMWt2QlhMOXpPdGs0SFMxT01jejIzMzhlRjNvRk9Db1BVSytQTDE0N0ZXNENBZWRhWmtLagpJTVRVc3Z4dW5LZk1CTEhQek0wQ0F3RUFBYU5aTUZjd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCCi93UUZNQU1CQWY4d0hRWURWUjBPQkJZRUZOY25DMVhjZG5xL1RtTCs0VWhRUVZsWG5UcnVNQlVHQTFVZEVRUU8KTUF5Q0NtdDFZbVZ5Ym1WMFpYTXdEUVlKS29aSWh2Y05BUUVMQlFBRXdnRUJBRGVOWm83UkdVWTJidFRZRVdFdwplYlFsUzh2WlBZWnhkUk9TR2I1cTNYTm5MTmNDNVA2TmJkT3Zkb1BIbDkzc2dFWFcxdnhvNWErTTF0WWp5dDB2CjZobkJxcFNydWhSVTVDVWFnYVRjcWNhMURkUWlMTzhuLm1uM003dHZiZ3BRYzJzY3hIdW10MUw4Zk9SdDg3YU8KTDNyMzN0VDhmNkFlL0xRVHY1RFBWcnJ1MHFveUtDdFA0YWRmS21TNW5tNXdUT2dqSDdkRHdtR3cyTFdlbDFGYgo2MUxJVWFKZFA0dXhXN3R2UExOUXBERmNKOUwxSSsyeHE2S2h2NGlFdjJEb3ArNHlLQjd6Z3cvbVdnOFhOS1M4Clc4cElxQzhFQmY5R0ZzMVUzbFpBMFFkWEVrNjRHWXlLL0lqb0tFTlBhaVRxSVNWNG5VQm82cW5YZUxtOEtTTFYKTlR3PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==",
+    "amazonResourceName": "arn:aws:eks:us-east-1:818164164244:cluster/test-cluster-1"
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/clusters/:cluster_id</a></code>
```

Comment:
Should it be `:id` instead of `:cluster_id`

Reply:
I believe I was going off of the GCP plugin API docs, it seems like this issue is also present on this [line](https://github.com/cloudops/cloudmc-api-docs/blob/d003ae9cee4ad30d18b6dfaf7f23b4b537fcc090/source/includes/gcp/_clusters.md?plain=1#L126), should I go ahead and make a change there as well or should I create a new ticket?